test/e2e/nightly/pvc_stress_fio/pvc_stress_fio_test.go

```diff
@@ -169,20 +201,48 @@ func TestPVCStress(t *testing.T) {
 	RunSpecs(t, "PVC Stress Test Suite")
 }
 
-var _ = Describe("Mayastor PVC Stress test with fio", func() {
-	It("should stress test use of PVCs provisioned over iSCSI and NVMe-of", func() {
-		stressTestPVC()
+var _ = Describe("Mayastor PVC Stress test", func() {
+	It("should stress test creation and deletion of PVCs provisioned over iSCSI and NVMe-of", func() {
+		stressTestPVC(cdIterations, false)
 	})
-})
+
+	It("should stress test creation and deletion of PVCs provisioned over iSCSI and NVMe-of", func() {
+		stressTestPVC(crudIterations, true)
+	})})
 
 var _ = BeforeSuite(func(done Done) {
-	logf.SetLogger(zap.LoggerTo(GinkgoWriter, true))
+	logf.SetLogger(zap.New(zap.UseDevMode(true), zap.WriteTo(GinkgoWriter)))
 
 	Cmn.SetupTestEnv()
+	tmp := os.Getenv("pvc_stress_cd_cycles")
```

Comment:
Other config envvars start with `e2e`, please let's keep that.

```suggestion
	tmp := os.Getenv("e2e_pvc_stress_cd_cycles")
```

Reply:
done.